solidity/test/token/ERC20WithPermit.test.js

```diff
@@ -1013,4 +1013,57 @@ describe("ERC20WithPermit", () => {
       })
     })
   })
+
+  describe("approveAndCall", () => {
+    const amount = initialSupply
```

Comment:
Can we use an amount different than `initialSupply` to make sure that the function uses the value passed in parameter instead of some value that is already there in the contract?

Reply:
https://github.com/keep-network/tbtc-v2/pull/9/commits/77bb248f613a502c15810ba0bcc08e982ee17bfe